strr-examiner-web/app/components/RegistrationDetails.vue

```diff
@@ -0,0 +1,40 @@
+<script setup lang="ts">
+
+const props = defineProps<{ application: ApiApplicationResp }>()
+
+const { header } = toRefs(props.application)
+
+</script>
+<template>
+  <ConnectPageSection
+    v-if="header?.registrationNumber"
+    :heading="{
+      label: 'Registration Details',
+      labelClass: 'font-bold',
+      icon: 'mdi-file-document-tick-outline'
```

Comment:
icon should have 'i-' prefix, so this should be 'i-mdi-file-document-tick-outline'

Reply:
Updated